lib/puppet/application/ssl.rb

```diff
@@ -23,13 +23,13 @@ def help
 ACTIONS
 -------
 
-* submit_request:
+* submit:
   Generate a certificate signing request (CSR) and submit it to the CA. If a private and
   public key pair already exist, they will be used to generate the CSR. Otherwise a new
   key pair will be generated. If a CSR has already been submitted with the given `certname`,
   then the operation will fail.
 
-* download_cert:
+* download:
```

Comment:
Shortening these is going to force us to make updates in the PE installer.

Reply:
It still accepts the old action names, the only difference is the help text...  but I can drop this commit?